src/java_bytecode/bytecode_info.h

```diff
@@ -1,3 +1,6 @@
+#ifndef CPROVER_JAVA_BYTECODE_BYTECODE_INFO_H
+#define CPROVER_JAVA_BYTECODE_BYTECODE_INFO_H
+
 #pragma once
```

Comment:
We should get rid of `#pragma once` (which is exactly why we are using header guards)